core/src/nodes/collection.rs

```diff
@@ -439,6 +439,17 @@ impl<TInEvent, TOutEvent, THandler> CollectionStream<TInEvent, TOutEvent, THandl
     }
 }
 
+/// Errors for interrupt(). 
+#[derive(Debug)]
+pub enum InterruptErr {
+    /// The task entry is vacant; it needs to be added first via add_reach_attempt
+    /// (with the TaskState set to Pending) before we try to connect.
+    VacantEntry,
```

Comment:
While it is technically correct that the problem is the entry in the hashmap being vacant, the real error is that they're trying to interrupt using an invalid reach attempt. I think this error variant should reflect that. How about `NoSuchReachAttemptId`? or just `ReachAttemptNotFound`?